### DISCUSSION TITLE
Winpilot 2024.7.1

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#229] [Announcements]
This release brings numerous code improvements and the ability to permanently pin the background color of Winpilot. With 'Refresh', you can initiate a color change, and use the 'Pin' button to save the color. False positives have been addressed in this release. I can't explain why all the antivirus programs are flagging it. JavaScript was injected into the web version without issues, and now harmless native .NET classes are triggering alerts. We'll see if things calm down.

## If you want an app reduced to the essentials, [otherwise go for XP-AntiSpy fork - XD-Antispy](https://github.com/builtbybel/xd-AntiSpy). I dug out and released the program a few days ago from the software archive

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/2024.7.1'>Winpilot 2024.7.1</a>.</em>

**Comment by b1zz1** [comment]
I don't know man... Maybe i'm too green on development world, but I definitely prefer Winpilot over the xd-AntiSpy and Bloatinosy. Sure, Winpilot itself looks like one of Microsoft's bloat apps, but this only make it better, also, it seems to work better and for now seems to have more functions, anyways, all of the (many) different verions are great!
